NGINXaaS for Azure: update CNI details on loadbalancer for kubernetes docs (#1337)
nginx/documentation: content/nginxaas-azure/loadbalancer-kubernetes.md

- Add the annotation: `nginx.com/nginxaas: nginxaas` to mark the service to be monitored by NLK.
- Choose one of the following `Service` types:
  - `NodePort`: To route external traffic into the cluster using a well defined port exposed on each AKS worker node.

>   - `ClusterIP`: To route traffic to pods directly if you are running an Azure Container Networking Interface (CNI) that lets you expose the pods on the Azure VNET. If you created a new AKS cluster through the AKS Extenstion installation, this should be automatically configured.

Reviewer: ```suggestion
  - `ClusterIP`: To route traffic to pods directly if you are running an Azure Container Networking Interface (CNI) that lets you expose the pods on the Azure VNET. If you create a new AKS cluster through the AKS Extension installation, it is configured automatically.
```
"should" is ambiguous (specially for ESL readers) and must be avoided if possible.

(my suggestion or any other option that avoids should)
